tasks/veekun.py

```diff
@@ -16,6 +19,32 @@
 
 @init_task_wrapper()
 async def csv_to_sqlite(conn: Connection) -> None:
+    latest_veekun_commit = ""
+    try:
+        latest_veekun_commit = (
+            subprocess.run(
+                ["git", "rev-list", "-1", "HEAD", "--", "data/veekun"],
+                capture_output=True,
+                check=True,
+            )
+            .stdout.decode()
+            .strip()
+        )
+        db = Database.open("veekun")
+        with db.get_session() as session:
+            db_commit = session.query(  # type: ignore  # sqlalchemy
+                v.LatestCommit.commit_id
+            ).scalar()
+            if db_commit == latest_veekun_commit:
+                return  # database is already up-to-date, skip rebuild
+
+    except (
+        subprocess.SubprocessError,  # generic subprocess error
+        FileNotFoundError,  # git is not available
+        OperationalError,  # table does not exist
+    ):
+        pass  # always rebuild on error
```

Comment:
We should probably add a generic log message after this code block or propagate the errors: a wrong setup keeps rebuilding the veekun tables silently.
This wouldn't be annoying since it's an occasional event, but it can still be implemented later after a logs refactor if you prefer.